scss/grid/_layout.scss

```diff
@@ -448,6 +448,14 @@
         }
     }
 
+    .k-edit-cell,
+    .k-grid-edit-row > td {
+        >.k-textbox,
+        >.k-widget,
+        > .text-box {
+            margin-left: -($cell-padding + $input-border-width);
```

Comment:
this will yield a "mixed unit error" when used by bootstrap or any other theme not using pixels as a base unit.